lib/userdata.inc.php.bak

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
dont add that to git :P

Reply:
Oh yes, that should absolutely not have been pushed. Fixed .gitignore in dd488106afd334734823824e0429b2d0b9f6ee81.